src/components/segments/VimeoVideo.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default class Video extends React.Component {
+  render() {
+    return (
+    	<div className="vimeo-video">
+      		<iframe src="https://player.vimeo.com/video/126984194" width="500" height="281" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```

Comment:
Seems like we'll want a responsive player. Here is a way to do that that I recently came across: http://jsfiddle.net/masau/7wrhm/